packages/react-sdk/src/core/components/StreamVideo/StreamVideo.tsx

```diff
@@ -0,0 +1,16 @@
+import {
+  StreamVideoProvider,
+  StreamVideoProps,
+} from '@stream-io/video-react-bindings';
+import { PropsWithChildren } from 'react';
+import { translations } from '../../../translations';
+
+export const StreamVideo = (props: PropsWithChildren<StreamVideoProps>) => {
+  const { children } = props;
+
+  return (
+    <StreamVideoProvider translationsOverrides={translations} {...props}>
+      {children}
+    </StreamVideoProvider>
+  );
+};
```

Comment:
```suggestion
export const StreamVideo = (props: PropsWithChildren<StreamVideoProps>) => {
  return <StreamVideoProvider translationsOverrides={translations} {...props} />;
};
```

I don't think there is a need to unwrap `children` and then wrap them again. `{...props}` forwards `children` to the provider as well.

Reply:
Done @oliverlaz, thanks for the suggestion. Never thought this way. 😄